### PR TITLE
Reworked patch #70 (termcolor optional)

### PR DESCRIPTION
**Pull request by stuertz**
This change improves and hopefully finishes the patch #70 from @gewthen 

* Made termcolor optional
* Added optional support for colorama
* fixed tests on windows (less options there!)

### CODE REVIEW

File: nosetimer/plugin.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
+# -*- mode: python; coding: utf-8 -*-
```
Comment (e0ne) [inline nosetimer/plugin.py]: Do we really need this?

File: nosetimer/plugin.py [hunk 2]
```diff
@@ -67,6 +81,18 @@ def empty(self):
...
 
+def _colorize(val, color):
+    """
+    colorize a string using termcolor or colorama, if any of them are available
+    """
+    if termcolor is not None:
+        return termcolor.colored(val, color)
+    if colorama is not None:
+        return TERMCOLOR2COLORAMA[color] + val + colorama.Style.RESET_ALL
+    else:
```
Comment (e0ne) [inline nosetimer/plugin.py]: nit: we can omit 'else' here

File: nosetimer/plugin.py (hunk 1 above)
Comment (stuertz) [reply nosetimer/plugin.py]: No, not really. Its a good practice in projects i work with. And reflects PEP 263.
I can remove it.

File: nosetimer/plugin.py (hunk 2 above)
Comment (stuertz) [reply nosetimer/plugin.py]: ok

**Comment by skudriashev** [comment]
Merged.

**Pull request closed without merging**